decred_wallet/Features/Wallet Setup/RecoverExistingWalletViewController.swift

```diff
@@ -34,6 +34,10 @@ class RecoverExistingWalletViewController: WalletSetupBaseViewController, UITabl
         registerObserverForKeyboardNotification()
         self.hideKeyboardWhenTappedAround()
         
+        //set border for dropdown list
+        self.wordSelectionDropDownContainer.layer.borderWidth = 1
+        self.wordSelectionDropDownContainer.layer.borderColor = UIColor(red:222/255, green:225/255, blue:227/255, alpha: 1).cgColor
```

Comment:
- Can this be translated to hex? It's simpler to read.
- Create a variable for this color in the `UIColor.swift` file to ensure that no other similar color exists already and that it isn't recreated subsequently if someone needs to use this same color.